product/timelines/miq_reports/tl_events_hourly.yaml

```diff
@@ -67,6 +68,7 @@ col_order:
 - container_namespace
 - container_name
 - container_replicator_name
+- middleware_name
```

Comment:
This is a slight departure from the way it's done in other places.  For example, above you can see several container_* entries.  And then analogous entries in timeline.rb (and also ems_event.rb).  I avoided hard-coding in those files, so that as we add more entities to the mw provider we don't need to keep remembering to update the .rb files.  Instead I just have a single entry here to handle the single entity object that may be provided for the event.  In the event pop-up it will have: "Middleware Resource: <resource-name-as-link>".  If you guys totally hate this approach I can change it to be hard-coded and require update as we go, but hopefully it's OK.